packages/react-components/react-skeleton/library/src/components/Skeleton/Skeleton.types.ts

```diff
@@ -9,6 +9,11 @@ export type SkeletonSlots = {
   root: NonNullable<Slot<'div'>>;
 };
 
+/**
+ * Sizes for the Skeleton
+ */
+export type SkeletonSize = 8 | 12 | 16 | 20 | 24 | 28 | 32 | 36 | 40 | 48 | 56 | 64 | 72 | 96 | 120 | 128;
```

Comment:
Maybe it would be good to import existing `SkeletonItemSize` or to import this type in `SkeletonItem.types.ts`.  Currently it duplicates `SkeletonItemSize`